pkg/frontend/mysql_cmd_executor.go

```diff
@@ -429,6 +429,34 @@ func getDataFromPipeline(obj interface{}, bat *batch.Batch) error {
 						row[i] = vs[rowIndex]
 					}
 				}
+			case types.T_decimal64:
+				fmt.Println("the scale for this decimal is", vec.Typ.Scale)
```

Comment:
Is this code left over from the development process?

Reply:
solved.